samples/C/main.c

```diff
@@ -0,0 +1,931 @@
+/*
+ * Geeqie
+ * (C) 2006 John Ellis
+ * Copyright (C) 2008 - 2012 The Geeqie Team
+ *
+ * Author: John Ellis
+ *
+ * This software is released under the GNU General Public License (GNU GPL).
```

Comment:
[Found one!](http://pastebin.com/tsd0ErxE) :D Or rather, edited one, after having cleaned up Google's crap code style.

GitHub: 1
GPL: 0

I AM THE THORN IN THE ARSE OF RICHARD STALLMAN.